x-pack/qa/sql/src/main/java/org/elasticsearch/xpack/qa/sql/jdbc/JdbcAssert.java

```diff
@@ -177,7 +178,12 @@ private static void doAssertResultSetData(ResultSet expected, ResultSet actual,
 
                     // handle nulls first
                     if (expectedObject == null || actualObject == null) {
-                        assertEquals(msg, expectedObject, actualObject);
+                        // hack for JDBC CSV nulls
+                        if ("null".equals(expectedObject)) {
```

Comment:
maybe check with use of lower or uppercase to allow `null` and `NULL`

Reply:
Done.